libraries/Microsoft.Bot.Builder.AI.LuisV3/LuisRecognizerOptions.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+using System.Net.Http;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.AI.Luis
+{
+    /// <summary>
+    /// Optional parameters for a LUIS recognizer.
+    /// </summary>
+     public class LuisRecognizerOptions
+    {
+        /// <summary>
+        /// Gets or sets the time in milliseconds to wait before the request times out.
+        /// </summary>
+        /// <value>
+        /// The time in milliseconds to wait before the request times out. Default is 100000 milliseconds.
+        /// </value>
+        public uint Timeout { get; set; } = 100000;
```

Comment:
Why not just make it `TimeSpan`? This would make it easier for the consumer to set instead of checking whether it's seconds or milliseconds, specially that the name doesn't indicate it.

Reply:
Done.